packages/libraries/cli/src/fragments/__.ts

```diff
@@ -0,0 +1 @@
+export * as Fragments from './_';
```

Comment:
needed? (both `_` and `__`)

Reply:
AFAIK Two modules is the only way to get a set module exports under one exported namespace. The names `_` and `__` could be anything. Do you know of a way to only need one module? And/or do you have alternate name suggestions?

Reply:
`index.ts`? I'm fine with `_` / `__`, no real strong opinions here.

Reply:
`index.ts` for `_` or `__`? And what do we call the other one (Or, perhaps you also feel code shouldn't be responsible for its own namespace in which case __ modules go away)?

In case its not clear: I'm keeping in mind that once we are using ESM internally `index` will no longer have any special behaviour and so imports will need to do `./a/b/c/index.js` rather than `./a/b/c`. That reduces advantage of `index`. One annoying thing about `index` is that in a directory its special status doesn't typically "rise to the top" of the directory listing yet its unique among all those modules and is better (IMO) locked in place consistently e.g. top/bottom whatever.

Having shared some of my thinking here, I'll stick with `_/__` since it sounds like you're open to it here, but also I want to continue to discuss this over time as we have more feelings and ideas about it too etc.